contracts/solver-registry/tests/test_withdraw_assets.rs

```diff
@@ -0,0 +1,266 @@
+use near_gas::NearGas;
+use near_sdk::NearToken;
+use serde_json::json;
+
+mod common;
+
+use common::utils::*;
+
+#[tokio::test]
+async fn test_withdraw_assets_from_pool() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
I suggest adding comments at the top describing what the test does. That makes it easier to understand and review